Runtime/Utils/UIBuildManager.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEngine.UIElements;
+
+namespace Mixin.Utils
+{
+    public class UIBuildManager<T> : Singleton<T> where T : Singleton<T>
```

Comment:
"where T : Singleton<T>"
Is this neccesary?

Reply:
i tried without but it didnt work. maybe you can have a try?

Reply:
public class UIBuildManager<T> : Singleton<<T>>

Reply:
`public class UIBuildManager : Singleton<UIBuildManager>`